### PR TITLE
Porting diagnostics to new repo

### PR DESCRIPTION
**Pull request by edcdavid**
This is to port the diagnostics functionality from the old repo. 
- Cni plugins are now retrieved for all nodes 
- Nodes summary is retrieved via goclient api (same as oc get nodes -ojson)
- Hw info is collected for each nodes, with the following functions:
  - lscpu -> lines to json
  - lsblk -> native json output
  - lspci -> native json output
  - ip a  -> native json output
- CSI driver details listed for the cluster 
The ability to run diagnostics by themselves without running a suite will be added in a future PR  (similar script/githubCI/doc changes as in https://github.com/test-network-function/test-network-function/pull/591)

### CODE REVIEW

File: cnf-certification-test/accesscontrol/rbac/automount_moq_test.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Code generated by moq; DO NOT EDIT.
```
Comment (hamadise) [inline cnf-certification-test/accesscontrol/rbac/automount_moq_test.go:1]: this file should not be here right ?

Comment (edcdavid) [reply cnf-certification-test/accesscontrol/rbac/automount_moq_test.go:1]: yep, will delete

Comment (edcdavid) [reply cnf-certification-test/accesscontrol/rbac/automount_moq_test.go]: fixed

Comment (sebrandon1) [reply cnf-certification-test/accesscontrol/rbac/automount_moq_test.go]: Probably needs a rebase and a re-run with the updated script from #73

**Review by greyerof — APPROVED** [review]
I would add the existing "non backward compatible" label to this PR, as it's changing the way the diagnostic info is retrieved and saved into the claim file. It is worth adding this change in the next release notes.

**Review by greyerof — APPROVED** [review]

File: internal/clientsholder/clientsholder.go [hunk 2]
```diff
@@ -134,6 +136,10 @@ func newClientsHolder(filenames ...string) (*ClientsHolder, error) { //nolint:fu
 	if err != nil {
 		logrus.Panic("can't instantiate ocClient", err)
 	}
+	clientsHolder.StorageClient, err = storageclient.NewForConfig(clientsHolder.RestConfig)
+	if err != nil {
+		logrus.Panic("can't instantiate csiClient", err)
```
[the remark this answers is not in the document]
Comment (edcdavid) [reply internal/clientsholder/clientsholder.go]: fixed

**Review by hamadise — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**